po/pt_BR.po

```diff
@@ -0,0 +1,353 @@
+msgid "hour"
+msgstr "hora"
+
+#: src/ui/project/record_create_window.ui:83
+#: src/ui/project/record_create_window.ui:142 src/ui/project/record_row.ui:45
+#: src/ui/project/record_row.ui:104
+msgid "minute"
+msgstr "minuto"
+
+#: src/ui/project/record_create_window.ui:101
+#: src/ui/project/record_create_window.ui:160 src/ui/project/record_row.ui:63
+#: src/ui/project/record_row.ui:122
+msgid "seconds"
+msgstr "segundos"
+
+#: src/ui/project/record_create_window.ui:121 src/ui/project/record_row.ui:83
+msgid "duration"
+msgstr "duração"
+
+#: src/ui/project/record_row.ui:142
+msgid "delete"
+msgstr "deletar"
+
+#: src/ui/project/task_page.ui:54 src/ui/project/task_page.ui:70
+msgid "Subtasks"
+msgstr "Subtarefas"
+
+#: src/ui/project/task_page.ui:75
+msgid "Records"
+msgstr "Registros"
+
+#: src/ui/project/task_page.ui:121
+msgid "no subtasks found"
+msgstr "nenhuma subtarefa encontrada"
+
+#: src/ui/project/task_page.ui:152
+msgid "no records found"
+msgstr "nenhum registro encontrado"
+
+#: src/ui/project/task_page.ui:153
+msgid "Use add button up there or start a timer"
+msgstr "Use o botão adicionar lá em cima ou inicie um temporizador"
+
+#: src/ui/search/search_window.ui:18
+msgid "Search"
+msgstr "Buscar"
+
+#: src/ui/search/search_window.ui:54
+msgid "no results found"
+msgstr "nenhum resultado encontrado"
+
+#: src/ui/sidebar/sidebar_projects.ui:14
+msgid "Projects"
+msgstr "Projetos"
+
+#: src/ui/sidebar/sidebar_projects.ui:33
+msgid "Show archive projects"
+msgstr "Mostrar projetos de arquivo"
+
+#: src/ui/sidebar/sidebar_projects.ui:40
+msgid "New project"
+msgstr "Novo projeto"
+
+#. Translators: Replace "translator-credits" with your names, one name per line
+#: src/application.rs:145
+msgid "translator-credits"
+msgstr "Fúlvio Leo <fulviokaldas@outlook.com>"
+
+#: src/views/window.rs:96 src/views/window.rs:244
+msgid "Personal"
+msgstr "Pessoal"
+
+#: src/views/window.rs:97 src/views/window.rs:245
+#: src/views/sidebar/sidebar_projects.rs:206
+msgid "Tasks"
+msgstr "Tarefas"
+
+#: src/views/project/project_done_tasks_window.rs:100
+msgid "Done Tasks"
+msgstr "Tarefas Concluídas"
+
+#: src/views/project/project_edit_window.rs:134
+msgid "Delete \"{}\" project?"
+msgstr "Deletar o projeto \"{}\"?"
+
+#: src/views/project/project_edit_window.rs:137
+msgid "The project and its tasks will be permanently lost."
+msgstr "O projeto e suas tarefas associadas serão perdidos permanentemente."
+
+#: src/views/project/project_header.rs:162
+msgid "Today"
+msgstr "Hoje"
+
+#: src/views/project/project_list.rs:361
+msgid "Delete \"{}\" list?"
+msgstr "Deletar a lista \"{}\"?"
+
+#: src/views/project/project_list.rs:363
+msgid "The list and its tasks will be permanently lost."
+msgstr "A lista e suas tarefas associadas serão perdidas permanentemente."
+
+#. Translators: {} Will be replaced with a shortcut label.
+#: src/views/project/project_lists.rs:60
+#, fuzzy
+msgid "Use the primary menu {} for adding new lists"
+msgstr "Use o menu principal"
```

Comment:
The format of this text changed. Now it uses `{}` for adding a shortcut label instead of splitting text.